src/platforms/android/configuration/integrations/jetpack-compose.mdx

```diff
@@ -8,13 +8,64 @@ categories:
   - mobile
 ---
 
-The `sentry-compose-android` library provides [Jetpack Compose](https://developer.android.com/jetpack/androidx/releases/compose) support for the following interactions:
-* User interactions like clicks, swipes and scroll gestures
-* Jetpack Compose navigation
+The `sentry-compose-android` library provides [Jetpack Compose](https://developer.android.com/jetpack/androidx/releases/compose) support in the following areas:
+* `@Composable` performance metrics
+* Transaction and breadcrumb support for User interactions (clicks, swipes and scroll gestures)
+* Transaction and breadcrumb support for Navigation events
+
+## Performance Metrics
```

Comment:
I'm thinking we should create a subpage per section, otherwise it's already a lot to read through. But no need to do it in this PR I guess, can be done later.